sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -514,12 +526,12 @@ def upsert_entity(  # pylint:disable=R1710
         """Update/Merge or Insert entity into table.
 
         :param entity: The properties for the table entity.
-        :type entity: Union[TableEntity, dict[str,str]]
+        :type entity: TableEntity or dict[str,str]
```

Comment:
side note: does this type correctly render in sphinx and lead you to the correct class? In sphinx I notice you sometimes fully qualify the class (i.e. `~azure.data.tables.TableEntity`) and sometimes don't, so want to ensure behavior's the same and they both work

Reply:
Yes, these ones map to the appropriate ones.